blog/en/2021_12_15.md

```diff
@@ -1,74 +1,48 @@
+### Dear ioBroker users,
 
-## Security in the network
-<!-- SOURCE: 761725 ## Sicherheit im Netz -->
-First of all, we would like to comment on the Log4j theme - some users have expressed concerns about the security of the system. But you can all breathe a sigh of relief, because the security gap in Log4j only affects Java systems and ioBroker is based on JavaScript - both languages have nothing in common, except for the first 4 letters.
-<!-- SOURCE: 535019 Erstmal möchten uns zum Theme Log4j äußern - einige User haben Bedenken zur Sicherheit des Systems geäußert. Ihr könnt aber alle aufatmen, denn die Sicherheitslücke bei Log4j betrifft nur Java Systeme und ioBroker basiert auf JavaScript - beide Sprachen haben nichts gemeinsam, außer die ersten 4 Buchstaben. -->
+By popular request we have decided to publish the results of our developer meeting here on the blog. In the future we will create at least one new entry every month. You can find the original meeting summary in german [here](https://forum.iobroker.net/topic/49502/meeting-f%C3%BCr-iobroker-core-dev-admin-15-12-21-20-30).
 
-But there is one problem that affects us all. Not just ioBroker, but all systems worldwide - namely port forwarding. It should be known, because we warn again and again that you shouldn't use port forwarding in order to be able to reach ioBroker or any other system while on the go. It is a shock to discover that users are using the systems openly in the network. Attackers can find such systems very easily and take control of the home automation. Please use a secure VPN connection, SSL and secure passwords and if you are not 100 %s ure what to do, please use our [secure cloud solution](https://iobroker.pro/www/). This will save you a rude awakening.
-<!-- SOURCE: 91768 Es gibt aber ein Problem, das uns alle betrifft. Nicht nur ioBroker, sondern alle Systeme weltweit - und zwar Portweiterleitungen. Es dürfte bekannt sein, denn wir warnen ja immer wieder davor, dass man keine Portweiterleitung nutzen soll, um ioBroker oder irgendwelche Systeme von unterwegs aus erreichen zu können. Mit Erschrecken muss man immer wieder feststellen, dass Anwender die Systeme ganz offen im Netz nutzen. Angreifer können solche Systeme sehr einfach finden und die Kontrolle der Hausautomatisierung übernehmen. Bitte nutzt eine gesicherte VPN Verbindung, SSL und sichere Passwörter und wenn ihr nicht 100%tig sicher seid was zu tun ist, dann benutze bitte unsere §§LLLLL_0§§. So bleibt euch ein böses Erwachen erspart. -->
+## Security on the net
+First of all, we would like to comment on the Log4j theme - some users have expressed concerns about the security of the system. But you can all breathe a sigh of relief, because the security hole in Log4j only affects Java systems and ioBroker is based on JavaScript - both languages have nothing in common except the first 4 letters.
```

Comment:
```suggestion
First of all, we would like to comment on the Log4j topic - some users have expressed concerns about the security of the system. But you can all breathe a sigh of relief, because the security hole in Log4j only affects Java systems and ioBroker is based on JavaScript - both languages have nothing in common except the first 4 letters.
```